### PR TITLE
Added question page and Results page

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: pages/question/question.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    font-size: 1.2rem;
+}
+.heading{
+    color: white;
+    text-align: center;
+}
+.quest{
+    color: var(--white);
+    margin: 2rem 0rem;
+    font-size: 1.25rem;
```
Comment (Hardiegogo) [inline pages/question/question.css:23]: I think you should use font sizes from your style guide to maintain consistency.

File: pages/question/question.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    background: rgb(35, 90, 35);
+}
+.red{
+    background: var(--red);
+}
+.next{
+    background: var(--grey);
+    width: max-content;
+    margin: 2rem 0rem;
+    box-shadow: 1px 1px 8px 2px #161616;
```
Comment (ArjunGTX) [inline pages/question/question.css:49]: You can store the shadow in a variable since it is being used in many occasions

**Review by ArjunGTX — APPROVED** [review]
Rest looks good to me

**Review by Isha2806 — APPROVED** [review]

**Review by Hardiegogo — COMMENTED** [review]
I have suggested a small change below.
Also you can remove your results.css file as it is empy.

File: pages/question/question.css (hunk 2 above)
Comment (shivanigangadharan) [reply pages/question/question.css:48]: Done [3af9b02](https://github.com/shivanigangadharan/quiz-master/pull/2/commits/3af9b02e8175955dc542680dd3f34e222dc00ecd)

File: pages/question/question.css (hunk 1 above)
Comment (shivanigangadharan) [reply pages/question/question.css]: Done [4aa0b5a](https://github.com/shivanigangadharan/quiz-master/pull/2/commits/4aa0b5ae446c686f9e786bbebb0a0077976b34f0)

Comment (shivanigangadharan) [reply pages/question/question.css]: Also removed result.css

**Pull request merged**